apps/indexer/ponder.schema.ts

```diff
@@ -104,17 +106,19 @@ export const delegation = onchainTable(
   }),
   (table) => ({
     pk: primaryKey({
```

Comment:
Same issue as delegation table - `transactionHash` is nullable but used in primary key. This will cause database errors when trying to insert records with null transaction hashes.